oscar.py

```diff
@@ -1537,132 +1548,116 @@ class Time_commit_info(Clickhouse_DB):
+        >>> rows = p.get_values_iter(['time','repo'], 1568571909, 1568571910)
         >>> for row in rows:
         ...     print(row)
         ...
         (1568571909, 'mrtrevanderson_CECS_424')
         (1568571909, 'gitlab.com_surajpatel_tic_toc_toe')
         (1568571909, 'gitlab.com_surajpatel_tic_toc_toe')
         ...
-        '''
+        """
         cols = self.__wrap_cols(cols)
-        rows_iter = self.query_select_iter(', '.join(cols), self.tb_name, start, end)
-        for row in rows_iter:
-            yield row
-    
-    def project_timeline(self, cols, project):
-        ''' return a generator for all rows given a project name (ordered by time)
-        >>> rows = p.project_timeline(['time','project'], 'mrtrevanderson_CECS_424')
+        return self.query_select_iter(', '.join(cols), self.tb_name, start, end)
+
+    def project_timeline(self, cols, repo):
+        """ return a generator for all rows given a repo name (ordered by time)
+        >>> rows = p.project_timeline(['time','repo'], 'mrtrevanderson_CECS_424')
         >>> for row in rows:
         ...     print(row)
         ...
         (1568571909, 'mrtrevanderson_CECS_424')
         (1568571909, 'mrtrevanderson_CECS_424')
         (1568571909, 'mrtrevanderson_CECS_424')
         ...
-        '''
+        """
         cols = self.__wrap_cols(cols)
-        query_str = 'SELECT {} FROM {} WHERE project=\'{}\' ORDER BY time'\
-                    .format(', '.join(cols), self.tb_name, project)
-        rows_iter = self.query_iter(query_str)
-        for row in rows_iter:
-            yield row
+        query_str = 'SELECT {} FROM {} WHERE repo=\'{}\' ORDER BY time'\
+                    .format(', '.join(cols), self.tb_name, repo)
+        return self.query_iter(query_str)
 
     def author_timeline(self, cols, author):
-        ''' return a generator for all rows given an author (ordered by time)
-        >>> rows = p.author_timeline(['time', 'project'], 'Andrew Gacek <andrew.gacek@gmail.com>')
+        """ return a generator for all rows given an author (ordered by time)
+        >>> rows = p.author_timeline(
+        ...     ['time', 'repo'], 'Andrew Gacek <andrew.gacek@gmail.com>')
         >>> for row in rows:
         ...     print(row)
         ...
         (49, 'smaccm_camera_demo')
         (677, 'smaccm_vm_hack')
         (1180017188, 'teyjus_teyjus')
         ... 
-        '''
+        """
         cols = self.__wrap_cols(cols)
         query_str = 'SELECT {} FROM {} WHERE author=\'{}\' ORDER BY time'\
                     .format(', '.join(cols), self.tb_name, author)
-        rows_iter = self.query_iter(query_str)
-        for row in rows_iter:
-            yield row
+        return self.query_iter(query_str)
 
     def __wrap_cols(self, cols):
-        ''' wraps cols to select before querying
-        '''
+        """ wraps cols to select before querying """
         for i in range(len(cols)):
-            if cols[i] == 'commit' or cols[i] == 'blob':
```

Comment:
the sha1 column no longer exits in the new table, the commit column is the replacement.